frontend/src/lib/components/LemonInput/LemonInput.tsx

```diff
@@ -105,7 +116,14 @@ export const LemonInput = React.forwardRef<HTMLInputElement, LemonInputProps>(fu
 
     return (
         <LemonRow {...rowProps}>
-            <input {...props} ref={textRef} />
+            <input {...props} ref={inputRef} />
         </LemonRow>
     )
 })
+
+export const LemonNumericInput = React.forwardRef<HTMLInputElement, LemonInputProps>(function _LemonInput(
+    props,
+    ref
+): JSX.Element {
+    return <LemonInput type="number" {...props} ref={ref} />
+})
```

Comment:
This wrapper doesn't seem super useful and provides two ways of doing one thing. Why not expose just `LemonInput`?